### ISSUE TITLE
Catamenial seizure

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10506]
**Preferred term label:**

**Synonyms**
Catamenial epilepsy
Menstrual-related epilepsy 

**Definition (free text, please give PubMed ID)**
PMID: 35484702 Catamenial pattern epilepsy is defined as an increase in the frequency of seizures during a specific stage of the menstrual cycle compared to baseline.

PMID: 34528245 Catamenial epilepsy describes worsening seizures in relation to the menstrual cycle and may affect around 40% of women with epilepsy

 PMID: 35205380 No seizure type or electroencephalographic pattern appears to be recurrent, although focal onset impaired awareness seizures (formerly defined complex partial seizures) might occur more frequently ; **catamenial seizures** have been noticed as well .

PMID: 20301487  Pubertal onset of **catamenial seizures** has also been observed in some females coinciding with menses 
[Smith & Gropman 2021]

**Parent term (use hpo.jax.org/app)**
Seizure HP:0001250

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Smith-Magenis syndrome ORPHA:819

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
I thiink it would be best to use a modifier such as Aggravated by menstruation HP:6000699
This is a bundled, disease-related phenotype and as such does not make a good HPO term.
